ui/shared/TransactionPreview.qml

```diff
@@ -362,11 +389,35 @@ Item {
                     height: 22
                     text: root.currency.toUpperCase()
                     color: Style.current.secondaryText
-                    anchors.right: parent.right
+                    anchors.right: gasArrow.visible ? gasArrow.left : parent.right
+                    anchors.rightMargin: gasArrow.visible ? Style.current.padding : 0
                     anchors.verticalCenter: parent.verticalCenter
                     horizontalAlignment: Text.AlignRight
                     verticalAlignment: Text.AlignVCenter
                 }
+                SVGImage {
+                    id: gasArrow
+                    width: 13
+                    visible: typeof root.gasClicked === "function"
+                    anchors.right: parent.right
+                    anchors.rightMargin: 7
+                    anchors.verticalCenter: parent.verticalCenter
+                    fillMode: Image.PreserveAspectFit
+                    source: "../app/img/caret.svg"
+                    rotation: 270
+                    ColorOverlay {
+                        anchors.fill: parent
+                        visible: parent.visible
+                        source: parent
+                        color: Style.current.secondaryText
+                    }
+                }
+                MouseArea {
+                    anchors.fill: parent
+                    visible: gasArrow.visible
+                    cursorShape: Qt.PointingHandCursor
+                    onClicked: root.gasClicked()
+                }
```

Comment:
Same here, looks like a `StatusIconButton`

Reply:
Unfortunately not, the design is for a much smaller button, and also the MouseArea covers more than just the button.